rho/translation.py

```diff
@@ -19,6 +19,7 @@
 def get_translation():
```

Comment:
not sure if this is still used because I see on other modules this import `from ansible.module_utils.translation import _`, maybe this module can be removed on a future iteration.